common/script_values/00_men_at_arms_values.txt

```diff
@@ -67,6 +67,12 @@ heavy_cavalry_recruitment_cost = @[heavy_cavalry_recruitment_cost]
 heavy_cavalry_low_maint_cost = @[heavy_cavalry_low_maint_cost]
 heavy_cavalry_high_maint_cost = @[heavy_cavalry_high_maint_cost]
 
+#Warcraft
+
+riflemen_recruitment_cost = 120
+riflemen_low_maint_cost = 0.4
+riflemen_high_maint_cost = @[riflemen_low_maint_cost * high_maint_mult]
```

Comment:
You refer to local `riflemen_low_maint_cost`. The local variables start with `@`. Thus the local `@riflemen_low_maint_cost` just doesn't exist and this doesn't work and reports an error.
```
[11:01:43][pdx_persistent_reader.cpp:421]: unknown variable [riflemen_low_maint_cost].  file: common/script_values/00_men_at_arms_values.txt line: 74
[11:01:43][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 74
```